palace/linalg/divfree.cpp

```diff
@@ -22,41 +22,45 @@ DivFreeSolver::DivFreeSolver(const MaterialOperator &mat_op, FiniteElementSpace
                              const std::vector<mfem::Array<int>> &h1_bdr_tdof_lists,
                              double tol, int max_it, int print)
 {
-  constexpr bool skip_zeros = false;
   MaterialPropertyCoefficient epsilon_func(mat_op.GetAttributeToMaterial(),
                                            mat_op.GetPermittivityReal());
   {
+    constexpr bool skip_zeros = false;
+    BilinearForm m(h1_fespaces.GetFinestFESpace());
+    m.AddDomainIntegrator<DiffusionIntegrator>(epsilon_func);
+    // m.AssembleQuadratureData();
```

Comment:
I'd like to leave these in (as with https://github.com/awslabs/palace/pull/167#discussion_r1465568493) in case eventually we do want to activate this (though for now it does appear to not be worth the extra memory overhead).